stable/meteor/values.yaml

```diff
@@ -38,6 +38,11 @@ resources:
 #         foo: "bar"
 #     sinks:
 #       - name: console
+cleanup:
+  enabled: false
+  image: curlimages/curl:8.9.1
```

Comment:
where this image will be used?

Reply:
Not used mas. Will delete it 👍  thank you